test/MSRIOGroupTest.cpp

```diff
@@ -83,11 +83,11 @@ class MSRIOGroupTest : public :: testing :: Test
         void mock_enable_fixed_counters(void);
 };
 
-class MockMSRIO : public geopm::MSRIOImp
+class MockForGroupMSRIO : public geopm::MSRIOImp
```

Comment:
I don't think I agree/understand the reason for these changes... We are worsening the coverage of the real MSRIOGroup class... No?

Reply:
This change is no longer necessary.  It was introduced because I had introduced a different object that I called a "MockMSRIO" in an earlier version.  I got rid of that mock in a newer revision, so this name can be changed back.